### PR TITLE
Dynamo storage enhancements

### PR DESCRIPTION
**Pull request by bigs**

### CODE REVIEW

File: transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala [hunk 1]
```diff
@@ -15,10 +15,12 @@ class DynamoDatastore(config: DynamoDatastore.Config)
   
   val creds = config.awscreds
   val table = config.baseTable
-  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit
+  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit incl field
```
Comment (vyzo) [inline transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala]: "... 400KB limit in item size" reads better for the comment

File: transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala [hunk 2]
```diff
@@ -15,10 +15,12 @@ class DynamoDatastore(config: DynamoDatastore.Config)
...
+                             // names
   val chunkTable = table + "Chunks"
-  
+
   val db = new AmazonDynamoDBClient(creds)
+  config.endpoint.foreach(ep => db.withEndpoint[AmazonDynamoDBClient](ep))
```
Comment (vyzo) [inline transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala]: [minor quip] just setEndpoint() -- why the fluid setter if you are discarding the value?

**Comment by vyzo** [comment]
just some minor quips -- :+1:

File: transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala (hunk 1 above)
Comment (bigs) [inline transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala]: i just wanted to be explicit that the actual names of the fields count towards the limit too i.e.

`meta`, `multihash` etc

**Comment by vyzo** [comment]
```
     .  o ..                  
     o . o o.o                
          ...oo               
            __[]__            
         __|_o_o_o\__         
         \""""""""""/         
          \. ..  . /          
     ^^^^^^^^^^^^^^^^^^^^  
```

**Pull request merged**